### PR TITLE
[14.0] [MIG] purchase_security

### PR DESCRIPTION
**Pull request by flook-chaiwat**
Standard Migration.

### CODE REVIEW

File: purchase_security/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * João Marques
```
Comment (kittiu) [inline purchase_security/readme/CONTRIBUTORS.rst:3]: You can add yourself as contributor if you want to (you can remark that you do the migration from v13 -> v14).

File: purchase_security/README.rst [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to **Purchase > Orders > Purchase Orders**
+#. Create a Purchase Order and assing a **Purchase Representative**
```
Comment (kittiu) [inline purchase_security/README.rst:44]:
```suggestion
#. Create a Purchase Order and assign a **Purchase Representative**
```

**Review by kittiu — APPROVED** [review]
Code review, no major change.

**Comment by kittiu** [comment]
@flook-chaiwat can you attend the comment.

**Review by fshah-initos — APPROVED** [review]
Code review and Functional Test. LGTM

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**